src/components/Header/header.scss

```diff
@@ -0,0 +1,211 @@
+        vertical-align: middle;
+        background: rgba(109, 107, 107, 0.15);
+        margin-right: 0px;
+        margin-left: auto;
+        border-radius: $buttonBorderRadius;
+       
+        @media (max-width: $mobile-breakpoint) {
+          display: grid;
+          align-items: center;
+          justify-content: right;
+          height: 40px;
+        }
+      }
+}
+    
+.logo {
+  display: flex;
+  align-items: center;
+  position: absolute;
+  left: 50%;
+  transform: translate(-50%, 0%);
+      
+  &:active, &:focus, &:hover{
+    text-decoration: none;
+  }
+        
+  &__textEvents {
+    font: $fontLogo;
+    color: $lightThemeAccept;
+
+    @media (max-width: $mobile-breakpoint) {
+      font-size: 14px;
+    }
+  }
+       
+  &__textExpress {
+    font: $fontLogo;
+    color:$white;  
+
+     @media (max-width: $mobile-breakpoint) {
+       font-size: 14px;
+     }
+  }   
+        
+    &__image {
+      height: 25px;
+
+       @media (max-width: $mobile-breakpoint) {
+         height: 15px;
+       }
+     }
+  }
+
+  .language-toggle {
+    display: flex;
+    margin-right: 2px;
+   
+    &__icon{
+      width: auto;
+      height: 16px;
+      color: $white;
+         
+     @media (max-width: $mobile-breakpoint) {
+       height: 11px;
+       margin-right: 1px;
+      }
+    }
+  }
+
+.header-icons {
+  display: flex;
+  vertical-align: middle;
+  justify-content: right;
+
+  &__item { 
+    width: auto;
+    height: 20px;
+    margin-right: 2px;
+
+      &:hover { 
+       height: 30px;
+      }
+
+    @media (max-width: $mobile-breakpoint) {
+      height: 12px;
+      margin-right: 1px;
+          }
+      }
+}
+
+.btn-light-theme {
+  padding: 2px 10px;
+
+  @media (max-width: $mobile-breakpoint) {
+    font-size: 10px;
+    padding: 1px;
+    justify-content: center;
+    margin-top: 2px;
+  }
+}
+
+.user-name{
+  margin: 0 3px 0 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  font-size: 16px;
+  color: $white;
+
+  @media (max-width: $mobile-breakpoint) {
+    font-size: 12px;
+    margin: 0;
+    }
+}
+.user-avatar{
+ 
+  @media (max-width: $mobile-breakpoint) {
+    visibility: hidden;
+    display: none;
+    }
+}
+
+.dropdown-toggle {
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  cursor: pointer;
+  color: $white;
+
+  @media (max-width: $mobile-breakpoint) {
+      margin-bottom: 2px;
+  }
+  }
+
+.dropdown-menu{
+  background-color: $white;
+  padding: 0;
+  margin-top: 10px;
+  border-radius: $buttonBorderRadius;
+
+  @media (max-width: $mobile-breakpoint) {
+    font-size: 10px;
+    margin-top: 1px;
+    }
+}
+.dropdown-item{
+  background-color: $white;
+  color: $lightThemeAccept;
+  margin: 2px;
+
+    &:hover{
+      padding-left: 50px;
+      border-bottom: 3px solid $lightThemeAccept;
+    }
+  
+}
+.removedecorations {
+  all:unset;
+
+   &:hover {
+     all:unset;
+   }
+}
```

Comment:
?

Reply:
class to remove formating in links. fixed formating